src/commands/importFunction/importFunctionApp.ts

```diff
@@ -0,0 +1,405 @@
+
+        const backendId = apiUtils.displayNameToIdentifier(funcAppName);
+        ext.outputChannel.appendLine(localize("importFunctionApp", `Create new backend ${backendId} for the function app...`));
+        await setAppBackendEntity(backendId, funcAppName, functionAppBase, backendCredentials, node);
+        for (const operation of allOperations) {
+            ext.outputChannel.appendLine(localize("importFunctionApp", `Create policy for operations ${operation.name}...`));
+            await node.root.client.apiOperationPolicy.createOrUpdate(node.root.resourceGroupName, node.root.serviceName, apiName, String(operation.name), {
+                format: "rawxml",
+                value: createApiOperationXmlPolicy(backendId)
+            });
+        }
+        ext.outputChannel.appendLine(localize("importFunctionApp", `Importing function app ${funcAppName} finished successfully!`));
+    }
+}
+
+function filteredExistingOperations(apiId: string, existingOperations: string[], allOperations: OperationContract[]): OperationContract[] {
+    const resOperations: OperationContract[] = [];
+    for (const operation of allOperations) {
+        const opName = String(operation.name);
+        if (existingOperations.includes(opName)) {
+            ext.outputChannel.appendLine(localize("importFunctionApp", `Resolving conflict operations... ${opName}`));
+            const appdNum = genUniqueOperationName(opName, existingOperations);
+            const nOperation: OperationContract = {
+                id: `${apiId}/operations/${opName}-${appdNum}`,
+                name: `${opName}-${appdNum}`,
+                displayName: `${operation.displayName}-${appdNum}`,
+                method: operation.method,
+                urlTemplate: `${operation.urlTemplate}-${appdNum}`,
+                description: "",
+                templateParameters: operation.templateParameters
+            };
+            resOperations.push(nOperation);
+        } else {
+            resOperations.push(operation);
+        }
+    }
+    return resOperations;
+}
+
+function genUniqueOperationName(opName: string, existingOperations: string[]): number {
+    let cnt = 0;
+    let curName = opName;
+    while (existingOperations.includes(curName)) {
+        cnt++;
+        curName = opName.concat("-", cnt.toString());
+    }
+    return cnt;
+}
+
+async function getAllOperations(node: ApiTreeItem): Promise<string[]> {
+    ext.outputChannel.appendLine(localize("importFunctionApp", `Get all operations from Api ${node.root.apiName}...`));
+    const operations: OperationCollection = await node.root.client.apiOperation.listByApi(node.root.resourceGroupName, node.root.serviceName, node.root.apiName);
+    const operationsNames: string[] = [];
+    operations.forEach(ele => {
+        operationsNames.push(String(ele.name));
+    });
+    return operationsNames;
+}
+
+async function getPickedFunctions(node: ApiTreeItem | ApisTreeItem, baseUrl: string): Promise<{
+    label: string;
```

Comment:
do we need the lable when we also get the IFunctionContract

Reply:
not really~